src/views/InitialScreen/services.vue

```diff
@@ -0,0 +1,56 @@
+<template>
+  <div class="service">
+    <div>
+      <h1>Services</h1>
+      <div class="data-view-wrapper">
+        <DataTable
+          :value="data"
+          striped-rows
+          responsive-layout="scroll"
+          :scrollable="true"
+          scroll-height="415px"
+        >
+          <Column field="year" header="Year"></Column>
```

Comment:
why is this static ?

Reply:
Its for demo purpose, services part need to be integrate